Tests/UnitTests/Identity/CustomerInfoManagerTests.swift

```diff
@@ -415,3 +378,129 @@ class CustomerInfoManagerTests: TestCase {
     }
 
 }
+
+// iOS 13.0+ only because these tests are async
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+class CustomerInfoManagerGetCustomerInfoTests: BaseCustomerInfoManagerTests {
```

Comment:
I think I covered all scenarios, but let me know if I missed any or if it'd be worth adding any more.